### ISSUE TITLE
Two Repo Install Instructions fail at METEOR_PACKAGE_DIRS

### ISSUE THREAD

**Issue by hammannja** [VulcanJS/Vulcan-Starter#2]
Using the Two-Repo Install Instructions at http://docs.vulcanjs.org/, when attempting Step #3, I get the following "not recognized" error:

PS C:\Users\Jerry\Documents\dGit\FairEnd> METEOR_PACKAGE_DIRS="C:/Users/Jerry/Documents/dGit/Vulcan/packages" meteor --port 3000 --settings settings.json

METEOR_PACKAGE_DIRS=C:/Users/Jerry/Documents/dGit/Vulcan/packages : The term 'METEOR_PACKAGE_DIRS=C:/Users/Jerry/Documents/dGit/Vulcan/packages' is not recognized
as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct
and try again.
At line:1 char:1
+ METEOR_PACKAGE_DIRS="C:/Users/Jerry/Documents/dGit/Vulcan/packages" m ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (METEOR_PACKAGE_...Vulcan/packages:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

I have verified that the path is correct, and also attempted "\\" instead of "/," without success.

**Comment by narigondelsiglo** [comment]
@hammannja the installing instructions applies to mac and linux (***bash***), as the command uses environment variables for ***bash***.
You should read how te set the meteor's METEOR_PACKAGE_DIRS option in windows, but this hasn't anything to do with VulcanJS.
Good luck!

**Comment by hammannja** [comment]
Thank you @narigondelsiglo! I was able to fix the problem as follows (in a Windows environment).

What I believe finally worked was entering in the terminal SET METEOR_PACKAGE_DIRS="~/Vulcan/packages", where "~" is replaced by a direct link to Vulcan/packages, which for me was "C:\Users\Jerry\Documents\dGit\Vulcan\packages."

Then, I restarted my computer.

Then I fixed some missing dependencies using "meteor npm install --save [missing dependencies identified by run script]." I believe there were two missing dependencies for me, the last of which was "cross-fetch."

Finally, it started working! Jerry
